oaipmh/tracks.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from django.db import transaction
+
+import mediaplatform.models as mpmodels
+
+
+LOG = logging.getLogger(__name__)
+
+
+#: Tags applied to media items created for lecture capture
+LECTURE_CAPTURE_TAGS = ['Lecture capture']
+
+
+@transaction.atomic
+def ensure_track_media_item(track):
+    # Nothing to do if the track already has a media item or if the associated series has no
+    # playlist
+    if track.media_item_id is not None:
+        return
+    if track.matterhorn_record.series is None:
+        return
+    if track.matterhorn_record.series.playlist_id is None:
+        return
+
+    if track.url == '' or track.url is None:
+        LOG.warn(
```

Comment:
this is prob a `LOG.error`, as this should never happened, otherwise something is going wrong with Opencast

Reply:
:+1: